Tests/Assetic/TwigFormulaLoaderTest.php

```diff
@@ -18,14 +18,17 @@ class TwigFormulaLoaderTest extends \PHPUnit_Framework_TestCase
     private $twig;
     private $activeTheme;
     private $logger;
+    private $resource;
+    private $resourceContent;
+    private $loader;
 
     public function setUp()
     {
         $this->twig = $this->prophesize('Twig_Environment');
         $this->activeTheme = $this->prophesize('Liip\ThemeBundle\ActiveTheme');
         $this->logger = $this->prophesize('Psr\Log\LoggerInterface');
         $this->resource = $this->prophesize('Assetic\Factory\Resource\ResourceInterface');
-        $this->resourceContent = 'test';
+        $this->resourceContent = new \Twig_Source('test', 'test');
```

Comment:
It's because builds some builds are using twig 2.x. before 2.x there were Deprecated error. That we should use \Twig_Source